base/sparse/sparsematrix.jl

```diff
@@ -1738,8 +1738,8 @@ for (Bsig, A1sig, A2sig, gbb, funcname) in
         global $funcname
         function $funcname(f::Function, B::$Bsig, A1::$A1sig, A2::$A2sig)
             func       = @get! cache  f  gen_broadcast_function_sparse($gbb, f, ($A1sig) <: SparseMatrixCSC)
```

Comment:
yeah, this style of method definition is supposed to be discourage now, with both staged functions and higher-order functions available for writing it better.